test/functional/interface_rpc.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Tests some generic aspects of the RPC interface."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class RPCInterfaceTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.setup_clean_chain = True
+
+    def test_batch_request(self):
+        self.log.info("Testing basic JSON-RPC batch request...")
+
+        request = [
+            # A basic requests that will work fine.
+            {"method": "getblockcount", "id": 1},
+            # Request that will fail.  The whole batch request should still
+            # work fine.
+            {"method": "invalidmethod", "id": 2},
+            # Another call that should succeed.
+            {"method": "getbestblockhash", "id": 3},
```

Comment:
nit, could use `getblockchaininfo` to avoid `getbestblockhash` below.

Reply:
Why would that be better / why should we "avoid" `getbestblockhash`?  A function that just returns a string seems simpler to me than one that returns an object (that's why I chose it), but it doesn't really matter.  So I'm happy to change it, but I don't see why that would be better?

Reply:
Just to save another call when it's irrelevant for this test.

Reply:
Exact, doesn't matter full expected result.

Reply:
Ok, I've changed the code now to just check that `result` is not `None` without checking what it actually is (so that no second RPC is needed for that).